assets/js/modules/analytics/components/settings/SettingsForm.js

```diff
@@ -79,6 +93,20 @@ export default function SettingsForm( {
 
 	return (
 		<Fragment>
+			{ ga4Enabled && ! ga4ModuleConnected && (
+				<SettingsNotice
+					type={ TYPE_WARNING }
+					LearnMore={ () => (
+						<Link href={ documentationURL } external>
+							{ __( 'Learn more', 'google-site-kit' ) }
+						</Link>
+					) }
+					notice={ __(
+						'Your current Universal Analytics will stop collecting data on July 1, 2023',
```

Comment:
I do wonder if this copy is quite correct, I'd have thought it would read better as follows:

```suggestion
						'Your current Universal Analytics property will stop collecting data on July 1, 2023',
```

I'll ask @marrrmarrr to comment on this one...

Reply:
@hussain-t, Mariya has confirmed this one on Slack, so please do go ahead and make the suggested change...

Reply:
Updated.